tests/Rector/Property/DoctrineTargetEntityStringToClassConstantRector/Fixture/Annotation/doctrine_string_target_entity_to_class.php.inc

```diff
@@ -96,6 +101,11 @@ final class MyEntity
      */
     private readonly ?Collection $items6;
 
+    /**
+     * @ORM\ManyToOne(targetEntity=\Rector\Doctrine\Tests\Rector\Property\DoctrineTargetEntityStringToClassConstantRector\Source\AnotherClass::class)
+     */
+    private readonly ?Collection $items7;
```

Comment:
Could you extract this test case into a new standalone test fixture file? Just so we have easier way to debug in case of future changes